### ISSUE TITLE
No diagnostic for empty finally-block triggered (Expected RCS1066)

### ISSUE THREAD

**Issue by Danghor** [JosefPihrt/Roslynator#673]
**Product and Version Used**: 
Roslynator 2019 Visual Studio Plugin 2.9.0

**Steps to Reproduce**:
Create a simple App using the following code

```csharp
try
{
    Console.WriteLine("Foo");
}
finally
{

}
```

**Actual Behavior**:
Roslynator should trigger RCS1066, since the finally statement is empty and therefore redundant.

**Expected Behavior**:
No diagnostic is triggered. RCS1066 is only triggered when a `catch`-block is introduced.
